### ISSUE TITLE
issue with the Progress Text = TIME option.

### ISSUE THREAD

**Issue by optyler** [LTmadness/FermenterPercentage#1]
Hello,

The `TIME` option which displays the time left seems to be bugged. 
It starts somewhere after 300 minutes... ^^' 

![image](https://user-images.githubusercontent.com/10259/154163910-bdf5bc41-4ea3-4588-b62c-0a0fcda65b1a.png)

Here are my confi.
Thanks
```
## Settings file was created by plugin Fermenter Percentage v0.0.6
## Plugin GUID: org.ltmadness.valheim.fermenterpercentage

[Advanced]

## Adds smooth color change to %/time
# Setting type: Boolean
# Default value: false
Use color = true

## Should the progress bar be off/red/yellow
# Setting type: ProgressBar
# Default value: OFF
# Acceptable values: OFF, RED, YELLOW
Progress Bar = RED

[General]

## Should the progress text be off/percent/time left
# Setting type: ProgressText
# Default value: PERCENT
# Acceptable values: OFF, PERCENT, TIME
Progress Text = TIME
```

**Comment by LTmadness** [comment]
Hey, thanks for letting me know, and sorry didn't see this earlier, will investigate and try to resolve ASAP

**Comment by LTmadness** [comment]
Fixed in newest version(1.0.0)

**Comment by optyler** [comment]
Thx, i'll try this soon :)
